### PR TITLE
Jira backup and restore functionality

### PR DESCRIPTION
**Pull request by corang**
with some minor cluster changes and a new zarf package jira backup and restore is now functional

### CODE REVIEW

**Comment by corang** [comment]
I swear I'm gonna get this CI to pass if it's the last thing I do

**Comment by anthonywendt** [comment]
I'm deploying and testing the backup restore process

File: backup-and-restore/jira/README.md
Comment (anthonywendt) [inline backup-and-restore/jira/README.md:1]: Should the docs live under docs. We have a backup-and-restore folder under docs as well. Was confusing when it didn't contain these docs I was expecting.

Comment (anthonywendt) [reply backup-and-restore/jira/README.md:1]: Maybe just update the relevant docs section to link and reference the appropriate README

Comment (corang) [reply backup-and-restore/jira/README.md:1]: > Maybe just update the relevant docs section to link and reference the appropriate README

That's what I'm thinking, just a small blurb about where the packages are and a small overview.

**Pull request merged**